src/components/ReorderableList.jsx

```diff
@@ -54,7 +54,10 @@ const ReorderableList = (props) => {
     <Button
       size='small'
       shape='circle'
-      disabled={currentPosition === 0}
+      disabled={
+        reorderableList[currentPosition].disabledReorder
```

Comment:
To make the ordering less messy the metadata tracks now work in this way: If disabled, a track option gets moved to the bottom along with any other metadata tracks and it can't be reordered, (it also makes more sense with the redux data we have).

To this end, I changed the `ReorderableList` so that it can receive a property that tells it to disable reordering for one item in particular (reordering's enabled by default so it doesn't affect other usage of this).